### PR TITLE
Suggestion for Docs structure

### PR DESCRIPTION
**Pull request by mfbenitezp**
Suggestion for Doc structure: I need to double check grammar, typos or links, but just to see if it is a clear structure. I tried to wear the shoes of an external researcher.

### CODE REVIEW

**Comment by HSalat** [comment]
Sorry, which is the new site and which is the old?

My point of view is the same as I mentioned on Slack. Make a very simple "table of contents" page and let the user decide what is important to them.

I would separate the troubleshooting for the simple reason that that way anyone looking at the site knows immediately where to click when they encounter a problem.

**Comment by dabreegster** [comment]
Old: https://alan-turing-institute.github.io/uatk-spc/
New: You can view the files in this PR, or more easily, 'gh pr checkout 21; cd docs; quarto preview' (assuming you've set up the github CLI and quarto)

File: docs/getting_started.qmd [hunk 1]
```diff
@@ -2,129 +2,21 @@
...
-repeat this sampling every day of the simulation, so they visit different
-venues. For primary and secondary school, it may be more appropriate to sample
-once and store that for the simulation -- a student probably doesn't switch
-schools daily.
+If all you need is to explore the enriched syntetic population file and mesuare if fits to you model, we have created the output file for **all counties in England**, so if the area you want to model is listed or you just want to explore the data, no need to run SPC yourself -- just [download data for your area](outputs.qmd).
 
-Alternatively, you can follow what ASPICS does. Every day, each person
-logically visits all possible venues, but their interaction there (possibly
-receiving or transmitting COVID) is weighted by the probability of each venue.
+If you are interested in create another area or combination of MSOAs, you will need then install and run SPC in your environment. We have tested SPC in Lunix and MacOS systems, for the moment there is no support for Windows systems, but we are working hard to address that.
```
Comment (dabreegster) [inline docs/getting_started.qmd:22]: Nobody's working on making this run in Windows, and I don't think there are plans to unless somebody needs it. I think most Windows devs wind up using [WSL](https://docs.microsoft.com/en-us/windows/wsl/install) or something else. As long as they can figure out how to run commands in a terminal and get a basic dev environment with cmake set up, there shouldn't be problems.

File: docs/getting_started.qmd [hunk 2]
```diff
@@ -2,129 +2,21 @@
...
-Note that `flows_per_activity` is stored in `InfoPerMSOA`, not `Person`. The
-flows for retail and school are only known at the MSOA level, not individually.
-So given a particular `Person` object, you first look up their household's MSOA
--- `msoa = population.households[ person.household ].msoa` and then look up
-flows for that MSOA -- `population.info_per_msoa[msoa].flows_per_activity`.
-
-Each person has exactly 1 flow for home -- it's just `person.household` with
-probability 1. A person has 0 or 1 flows to work, based on the value of
-`person.workplace`.
+SPC creates Google's [protocol buffer files](https://developers.google.com/protocol-buffers/docs/overview) a platform-neutral, extensible mechanism for serializing structured data in bi-directional way (forward-compatible and backward-compatible). SPC encodes the enriched synthetic population [schema](https://github.com/alan-turing-institute/uatk-spc/blob/main/synthpop.proto) in .proto files (.pb). You can read the "protobuf" (shorthand for a protocol buffer file) in any
```
Comment (dabreegster) [inline docs/getting_started.qmd:13]: From the outputs page: "Over time, we may add more data to SPC or change the schema. Protocol buffers are designed to let combinations of new/old code and data files work together, but we don’t intend to use this feature. We may make breaking changes, like deleting fields. We’ll release a new version of the schema and output data every time and document it here. You should depend on a specific version of the data output in your code, so new releases don’t affect you until you decide to update."

The forward/backward compatible aspect of protobufs is explicitly not used in this project, I wouldn't mention it at all

File: scripts/select_msoas.py [hunk 3]
```diff
@@ -37,6 +37,5 @@ def area(lad):
             f"No matches for LAD20NM = {lad} in data/raw_data/referencedata/lookUp.csv"
         )
 
-
```
Comment (dabreegster) [inline scripts/select_msoas.py:40]: Why's there a change here? Running the black formatter puts this newline back

File: docs/getting_started.qmd (hunk 1 above)
Comment (mfbenitezp) [reply docs/getting_started.qmd:22]: Ok.

File: docs/getting_started.qmd (hunk 2 above)
Comment (mfbenitezp) [reply docs/getting_started.qmd:13]: Ok

File: scripts/select_msoas.py (hunk 3 above)
Comment (mfbenitezp) [reply scripts/select_msoas.py:40]: No idea. Ignore it.

**Pull request merged**